Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1339,6 +1367,11 @@ extension BrowserViewController : Transitionable {
             }
         }
         self.homePanelController?.view.hidden = false
+        if options.toView === self {
+            startTrackingAccessibilityStatus()
+        } else {
+            stopTrackingAccessibilityStatus()
+        }
```

Comment:
I was nervous too with discovering the various places to put the tracking calls at. I however did not get what you meant by overriding `UIAccessibilityIsVoiceOverRunning` and how does that relate to KIF tests?